### DISCUSSION TITLE
Filtering with cut and paste no longer works in tables.

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#589] [PrimeFaces] [answered]
Since PF 13.0.2 filtering datatable with cut and paste does not longer work. Sometimes it works, but most of the time it doesn't. In PF 13.0.1 filtering works as expected. I can see that filtering works fine in your show case. But the show case just uses PF 13.0.0 and not the current version 13.0.3. 

Is there any workaround for us to fix that problem? Will that be fixed in on of the next versions?

Thank you

**Comment by jepsar** [comment]
Please [create an issue](https://github.com/primefaces/primefaces/issues/new/choose) and include a [PrimeFaces Test reproducer](https://github.com/primefaces/primefaces-test)

**Reply by melloware** [reply]
Also here are all the fixes between 13.0.1 and 13.0.2 and nothing stands out to me as what would be related to filtering or cut and paste?

https://github.com/primefaces/primefaces/issues?q=is%3Aclosed+milestone%3A13.0.2

**Reply by hgfrey** [reply]
I will try to create a test reproducer, but currently I'm not able to download the jetty maven plugin. But I'll stick to that topic.

**Comment by hgfrey** [comment]
I have added a table that is as simple and filterable as possible to your primefaces test. Little by little I added various elements from our application. It feels like filtering via Ctrl-v becomes less reliable from time to time. With version 13.0.1 the filter works reliably. With 13.0.2 the filter sometimes works and sometimes it doesn't work. I have packaged and attached my version. We are using openJjdk - 11. 

Thanks for the help.

[primefaces-test-master.zip](https://github.com/primefaces/community/files/13648393/primefaces-test-master.zip)

**Reply by melloware** [reply]
Your download is incorrect the zip is totally just the original PF test project???

**Reply by hgfrey** [reply]
oops...

**Reply by hgfrey** [reply]
Sorry. I've grabbed the wrong zip file.

[primefaces-test-hgfrey.zip](https://github.com/primefaces/community/files/13649652/primefaces-test-hgfrey.zip)

**Reply by melloware** [reply]
Seems to be working OK for me I can keep cutting and pasting into the filter over and over?

**Reply by hgfrey** [reply]
it seems to me that this may be a timing problem. Our virtual machines are very slow. :-( 

Sometimes it works and sometimes it doesn't.

I'll try adding more elements.

**Reply by melloware** [reply]
OK I am on a very fast machine.  I wonder if its related to the filterDelay?  have you tried filterDelay=0 ?

**Comment by hgfrey** [comment]
In this version I've added two more event listeners. Now in most cases the filter no longer works with cut and paste. Selecting the content of a filter with Ctrl-A and removing it with Ctrl-X also very often doesn't work as expected. In fact, not at all.

[primefaces-test-hgfrey-2.zip](https://github.com/primefaces/community/files/13674881/primefaces-test-hgfrey-2.zip)

**Reply by melloware** [reply]
OK I see the issue.  Debugging.

**Reply by melloware** [reply, marked as answer]
https://github.com/primefaces/primefaces/issues/11125

**Reply by melloware** [reply]
This will be fixed in 13.0.4
